CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -64,7 +64,7 @@ public BasicDataProvider(Account currentAccount)
                 await jobs.ForEachAsync(item => jobModels.Add(new JobModel(item)));
             }
 
-            return jobModels;
+            return jobModels.OrderByDescending(t => t.CreationTime).ToList();
```

Comment:
I'm concerned about doing this for jobs but not for pools/tasks/job schedules.

Is this required, or should we just add a field in the table for job creation time and allow you to click on that to sort by creation time if you want?